### ISSUE TITLE
Acess Plan editor: Text mode still acts as Visual mode upon creation

### ISSUE THREAD

**Issue by eri-trabiccolo** [gocodebox/lifterlms#1562]
### Reproduction Steps

+ Create an access plan
+ Write something in the description in Visual mode and change the text formatting, e.g. make something bold. This is to better see what happens, it's not needed.
+ Click on the Text tab

### Expected Behavior

+ I'd expect to see the HTML of what I produced in visual mode.

### Actual Behavior

+ I still see the visual mode, additionally the Text mode toolbar is visible but not functional
![Schermata 2021-03-15 alle 12 34 52](https://user-images.githubusercontent.com/7689242/111147676-0671c600-858b-11eb-8ad2-74fdfe1c3a83.png)

If you save the access plan the Text mode then displays the HTML but the toolbar isn't visible.
Refreshing the page solves the issue. So let's say this is the workaround.

### Error Messages / Logs

```
n/a
```

### System and Environment Information

```
LifterLMS 4.19.0
WordPress 5.7.0
Theme: Twenty Twentyone
```

This issue has be recreated:
+ [x] Locally
+ [ ] On a staging site
+ [ ] On a production website
+ [x] With only LifterLMS and a default theme

### Browser, Device, and Operating System Information

n/a

### Related User Information
Nobody reported it as far as I know.

**Comment by thomasplevy** [comment]
@eri-trabiccolo any idea if this is a regression/new bug introduced by #1555 or has it "always" (or for a while) worked like this and we just haven't noticed? I'm not sure it truly matters but I'm just curious if we should start with the new code from #1555 as a source of this problem or not.

**Comment by eri-trabiccolo** [comment]
@thomasplevy 
I don't think this is a regression. It's unrelated to me. It wasn't always so, it might be due to some change in wp.

**Comment by thomasplevy** [comment]
@eri-trabiccolo gotcha.

For now, I think let's let this hang with the "alternative" you've suggested and figure out a way to fix it in the future when we have capacity or requirements to do so.
